src/robusta/core/sinks/slack/templates/template_loader.py

```
@@ -0,0 +1,125 @@
+            context: Dictionary of variables to pass to the template
+        Returns:
+            List of Slack block objects (dictionaries)
+        """
+        try:
+            template = Template(custom_template)
+            return self.render_to_blocks(template, context)
+        except Exception as e:
+            logging.error(f"Error rendering custom template: {e}")
+            return []
+
+    def render_file_template_to_blocks(self, template_name: str, context: Dict[str, Any]) -> List[Dict[str, Any]]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)